junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -137,4 +137,16 @@
 	@API(status = EXPERIMENTAL, since = "5.6")
 	String[] nullValues() default {};
 
+	/**
+	 * The maximum characters of per CSV column allowed.
+	 *
+	 * <p>To enable auto-expansion of the internal array, set this property to -1
```

Comment:
**Team Decision**: Let's not support the `-1` feature of the underlying parser. Instead, let's document that the configured value must be positive and validate that.

Reply:
@sbrannen I have seen the comments newly, yeah i have time to update pr according to comments. I will check it.

Reply:
Thanks for review. Validation is added.